plugins/modules/simpleinit_msb.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016-2023, Vlad Glagolev <scm@vaygr.net>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: simpleinit_msb
+short_description: Manage services on Source Mage GNU/Linux
+version_added: 7.1.0
+description:
+    - Controls services on remote hosts using C(simpleinit-msb).
+author: "Vlad Glagolev (@vaygr)"
+extends_documentation_fragment:
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
+options:
+    name:
+        type: str
+        description:
+            - Name of the service.
+        required: true
+        aliases: ['service']
+    state:
+        type: str
+        required: false
+        choices: [ running, started, stopped, restarted, reloaded ]
+        description:
+          - V(started)/V(stopped) are idempotent actions that will not run
+            commands unless necessary.  V(restarted) will always bounce the
+            service.  V(reloaded) will always reload.
+          - At least one of O(state) and O(enabled) are required.
+          - Note that reloaded will start the
+            service if it is not already started, even if your chosen init
+            system would not normally.
+    enabled:
+        type: bool
+        required: false
+        description:
+        - Whether the service should start on boot.
+        - At least one of O(state) and O(enabled) are required.
```

Comment:
BTW, you are using three different indent levels for lists: in most cases you use four spaces, for `state`'s description two spaces, and here zero spaces. This isn't a problem, but I would probably switch to four (or two) everywhere.

(Definitely keep two spaces in `EXAMPLES`.)

Reply:
Moved to 2 everywhere, thanks.